connectors/email/src/test/java/io/camunda/connector/email/client/jakarta/JakartaExecutorTest.java

```diff
@@ -75,7 +75,77 @@ void executeSmtpSendEmail() throws MessagingException {
                   try {
                     return Arrays.stream(argument.getFrom())
                             .allMatch(address -> address.toString().contains("myself"))
-                        && argument.getContent().toString().contains("body");
+                        && argument.getContent().toString().contains("body")
+                        && argument.getContentType().contains("text/plain");
+                  } catch (MessagingException | IOException e) {
+                    throw new RuntimeException(e);
+                  }
+                }),
+            argThat(
+                argument ->
+                    Arrays.toString(argument).contains("to")
+                        && Arrays.toString(argument).contains("cc")
+                        && Arrays.toString(argument).contains("bcc")));
+  }
+
+  @Test
+  void executeSmtpSendEmail() throws MessagingException {
+    buildSmtpTest(ContentType.PLAIN, "body", "text/plain");
+  }
+
+  @Test
+  void executeSmtpSendEmailAsHtml() throws MessagingException {
+    buildSmtpTest(ContentType.HTML, "<html><body>body</body></html>", "text/html");
+  }
+
+  @Test
+  void executeSmtpSendEmailAsMultiPart() throws MessagingException {
+    buildSmtpTest(ContentType.MULTIPART, "<html><body>body</body></html>", "multipart/mixed");
```

Comment:
How to add the second part (in plain text) to the email body?

Reply:
I believe we cant do it, I am working on making it work now @ingorichtsmeier